pcdsdevices/attenuator.py

```diff
@@ -1125,6 +1324,29 @@ class AT1K4(AttenuatorSXR_Ladder):
     """
 
 
+class AT1K2(AttenuatorSXR_LadderTwoBladeLBD):
+    """
+    AT1K2 solid attenuator variant from the LCLS-II L2SI project.
+
+    This has 2 blades, each with up to 8 filters each.
+    This class includes a calculator to aid in determining which filters to
+    insert for a given attenuation at a specific energy.
+    This class also includes control for the Kurt J. Lesker LBD system,
+    allowing IN/OUT state for mirror mount.
+
+    Parameters
+    ----------
+    prefix : str
+        Solid Attenuator base PV.
+
+    name : str
+        Alias for the Solid Attenuator.
+
+    calculator_prefix : str
+        The prefix for the calculator PVs.
+    """
```

Comment:
Would it be possible to add this to the test suite fixture? I'm not actually sure how this works but it seems like we're somehow (successfully?) testing that the move commands work for the other attenuators.

https://github.com/pcdshub/pcdsdevices/blob/b66e6808a58679b1ed3cbb906ec6821b131acaf5/pcdsdevices/tests/test_attenuator.py#L171-L181

Reply:
😢

Reply:
@ZLLentz Ah! Missed that. Submitting a new PR for this